static/app/views/alerts/rules/metric/types.tsx

```diff
@@ -95,6 +102,7 @@ export type Trigger = Partial<SavedTrigger> & UnsavedTrigger;
 export type UnsavedMetricRule = {
   aggregate: string;
   dataset: Dataset;
+  detectionType: string;
```

Comment:
can we use the `AlertRuleComparisonType` enum for this instead of a string? (or if that's already available, do we need to add this?)

Reply:
detectionType needs to be a string because this is what's passed into the API

Reply:
Or, the strings for the model are slightly different from the `AlertRuleComparisonType` strings, so I have a map that translates between the two and sets detectionType to be a string